### ISSUE TITLE
FireFox: close icon tooltip inside modal component is not show 

### ISSUE THREAD

**Issue by jingxxue** [IBM/carbon-components-react#1964]
## Environment

> Operating system : WINDOWS 10

> Browser : FIREFOX 65.0.2

> Automated testing tool and ruleset

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?

    "carbon-components": "^9.82.13",
    "carbon-components-react": "^6.99.1",
    "carbon-icons": "^7.0.7",

> What did you expect to happen?
when hoover on the close icon inside modal, Tooltip with "close modal" should be show
> What happened instead?
not tooltip show when hoover on the close icon
> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one 
open a modal
2. Step two
hoover on the close icon on top right corner
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/codesandbox

## Additional information

- Screenshots or code
- Notes
in the new version of firefox, Firefox ignore the tittle element inside the svg, if the svg element is inside a button element, suggest to move the title as a attribute of button.

**Comment by jingxxue** [comment]
the codeSandBox test case:  https://codesandbox.io/s/526moz78ok

**Comment by dakahn** [comment]
Looks like the aria-label is being set properly. Cross checked the bug on Chrome as well and yeah, there's no tool tip there either. Wondering if this is a recent regression -- did this work in the past?

**Comment by jingxxue** [comment]
> Looks like the aria-label is being set properly. Cross checked the bug on Chrome as well and yeah, there's no tool tip there either. Wondering if this is a recent regression -- did this work in the past?

is working in previous version of FireFox, but not the new version

**Comment by vpicone** [comment]
I believe the issue is the SVG doesn't take up the entirety of the button. So the outside edges of the button are clickable but do not show the SVG title tooltip.

In the test case, if you hover over the center of the icon, you'll see the tooltip show up.

**Comment by dakahn** [comment]
Thats a great catch @vpicone!

**Comment by vpicone** [comment]
Working on a PR now :)

**Comment by jingxxue** [comment]
> I believe the issue is the SVG doesn't take up the entirety of the button. So the outside edges of the button are clickable but do not show the SVG title tooltip.
> 
> In the test case, if you hover over the center of the icon, you'll see the tooltip show up.
